kde-apps/kde-l10n/kde-l10n-16.12.0.ebuild

```diff
@@ -37,6 +37,7 @@ RDEPEND="
 	!kde-apps/ktp-l10n
 	!kde-apps/ktp-send-file:4
 	!kde-apps/ktp-text-ui:4
+	!<=kde-apps/kwave-0.9.2
```

Comment:
Yes, but it indicates when the proj was added to kde apps and makes blocker cleanup easier.